data/munin/groonga_n_records_

```diff
@@ -202,15 +237,17 @@ def autoconf
 end
 
 def suggest
-  if @http_host and @http_port
+  exit(false) unless pid_files_esists?
+
```

Comment:
Do we need this?

Reply:
No, this is needless.
I've deleted this.